server/src/main/scala/com/cloudera/livy/LivyConf.scala

```diff
@@ -42,6 +42,13 @@ object LivyConf {
   val LIVY_SPARK_MASTER = Entry("livy.spark.master", "local")
   val LIVY_SPARK_DEPLOY_MODE = Entry("livy.spark.deployMode", null)
 
+  // Two configurations to specify Spark and related Scala version. These are internal
+  // configurations will be set by LivyServer and used in session creation. It is not required to
+  // set usually unless running with unofficial Spark + Scala versions
+  // (like Spark 2.0 + Scala 2.10, Spark 1.6 + Scala 2.11)
+  val LIVY_SPARK_SCALA_VERSION = Entry("livy.spark.scalaVersion", null)
+  val LIVY_SPARK_VERSION = Entry("livy.spark.version", null)
```

Comment:
I've given these configs more thought. I think we should remove livy.spark.version because we are always detecting the version from spark-submit. And we are using require() to make sure they match. I don't think this LivyConf is useful for customers.
Is this added to pass spark version and scala version to InteractiveSession?

If you are too busy to make the change, I can do it in #211.